site2/website/versioned_docs/version-2.7.1/getting-started-helm.md

```diff
@@ -81,9 +81,14 @@ We use [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) i
 
 3. Use the Pulsar Helm chart to install a Pulsar cluster to Kubernetes.
 
+   > NOTE: Please specify `--set initialize=true` when installing a release at the first time. `initialize=true` will start initialize jobs
+   >       to initialize the cluster metadata for both bookkeeper and pulsar clusters.
```

Comment:
> **NOTE**  
> You need to specify `--set initialize=true` when installing Pulsar the first time. Setting `initialize=true` starts initializing jobs to initialize the cluster metadata for both BookKeeper and Pulsar clusters.

Reply:
When you write "starts initializing jobs to initialize", do you mean "Setting `initialize=true` starts initializing the cluster metadata....

Reply:
Reference from https://github.com/apache/pulsar-helm-chart
![image](https://user-images.githubusercontent.com/30121694/103220744-01be4e00-495c-11eb-836e-e1987dece780.png)

Reply:
@yufan022 Thanks for your reference. I think "This command installs and starts Apache Pulsar" is clear and concise. Maybe it's better to use this one. If you want to add BK, you can refine it as "This command initializes the metadata for both BookKeeper and Pulsar clusters."

Reply:
You mean change 
`NOTE: Please specify --set initialize=true when installing a release at the first time. `initialize=true` will start initialize jobs to initialize the cluster metadata for both bookkeeper and pulsar clusters.`
to
`This command installs and starts Apache Pulsar` ？

Reply:
I mean you can change it into 
```
> **NOTE**  
> You need to specify `--set initialize=true` when installing Pulsar the first time. This command installs and starts Apache Pulsar.
```
or
```
> **NOTE**  
> You need to specify `--set initialize=true` when installing Pulsar the first time. This command initializes the metadata for both BookKeeper and Pulsar clusters.
```